src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,99 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final int CINEMA_HALL_CAPACITY = 50;
+    private static final int FAST_AND_FURIOUS_MOVIE = 0;
+    private static final int THE_GODFATHER_MOVIE = 1;
+    private static final int THE_DARK_KNIGHT_MOVIE = 2;
+    private static final int CINEMA_HALL_NUMBER_ONE = 0;
+    private static final int CINEMA_HALL_NUMBER_TWO = 1;
+    private static final int CINEMA_HALL_NUMBER_THREE = 2;
+    private static final LocalDate DATE_24_12_25 = LocalDate.of(2024, 12, 25);
+    private static final LocalDate DATE_24_12_27 = LocalDate.of(2024, 12, 27);
+    private static final int MOVIE_SESSION_ONE = 0;
+
     public static void main(String[] args) {
-        MovieService movieService = null;
 
-        Movie fastAndFurious = new Movie("Fast and Furious");
-        fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
-        movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
+        // 1. MovieService realisations (add, get, getAll).
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
+        List<Movie> movies = createSampleMovies();
+
+        for (Movie movie : movies) {
+            movieService.add(movie);
+        }
+
+        System.out.println(movieService.get(movies.get(FAST_AND_FURIOUS_MOVIE).getId()));
```

Comment:
Directly calling `.get()` on an `Optional` without checking if a value is present can lead to `NoSuchElementException`. Consider using `ifPresent` or `orElseThrow` to handle absent values more gracefully.